### DISCUSSION TITLE
All Google Takeout Uploads going to Admin User Regardless of API Key

### DISCUSSION THREAD

**Discussion by rynh** [immich-app/immich#8305] [Q&A]
Having a strange thing happening ... I first uploaded my google takeout zip to the admin account using a generated API key.

I'm now trying to upload a second set of takeout zips to a separate users account, with their own API key ... but that import is ending up in the admin account still. This is the command I'm using. Does anyone see anything wrong with this? Do the order of the operators matter?

`immich-go -server http://x.x.x.x:port -key APIKEY -log-file=log.txt upload -google-photos -create-albums=true Z:\Downloads\user-google-photos\takeout-*.zip`

**Comment by mmomjian** [comment]
Syntax looks correct to me. Try posting the issue [here](https://github.com/simulot/immich-go)

**Reply by mmomjian** [reply]
As of 0.13, there is a new feature to store the API key at `$HOME/.immich-go/immich-go.json`. My guess is there's some kind of bug/issue with that.

**Comment by bo0tzz** [comment]
Closing since this is likely an issue in immich-go

**Reply by rynh** [reply]
Sorry about that, in the wrong repo! Ugh
